metadata-integration/java/openlineage-converter/src/test/java/io/datahubproject/openlineage/HdfsPathDatasetTest.java

```diff
@@ -121,7 +121,7 @@ public void testPathSpecListPlatformInstance()
                         "s3",
                         Collections.singletonList(
                             PathSpec.builder()
-                                .env("PROD")
+                                .env(Optional.of("PROD"))
```

Comment:
This test was not compiling running as part of the complete test run because it was junit instead of testng